src/main/java/graphql/language/NodeChildrenContainer.java

```diff
@@ -0,0 +1,96 @@
+package graphql.language;
+
+import graphql.PublicApi;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import static graphql.Assert.assertNotNull;
+
+@PublicApi
+public class NodeChildrenContainer {
```

Comment:
JavaDoc?

Maybe a seperate PR for the JavaDoc say

Reply:
added